frontend/src/components/SlideSearchFunction.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import "./SlideSearchFunction.css";
+import GenreList from "./GenreList";
+
+class SlideSearchFunction extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      searching: true
+    };
+  }
+  render() {
+    return (
+      <div>
+        <button //button a placé dans le top bar
```

Comment:
- Comments in English, please
- If you know what should be done, please do it or alert another dev for her/him to do it

Reply:
comments is del, and I add a sticky on trello